ports/fp16/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "fp16",
+  "version-string": "2021-02-21",
+  "description": "Conversion to/from half-precision floating point formats",
```

```suggestion
  "description": "Header-only library for conversion to/from half-precision floating point formats",
```
It would be better to add `header-only library` here.

---

Applied the suggestion in 4cdaa9d.